### PR TITLE
Week4 checkpoint1

### PR DESCRIPTION
**Pull request by CesarRiv**
Was successful on assigning the test cases to a variable and constructing a sorted list of times that are already occupied in the calendar.

### CODE REVIEW

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -15,9 +15,45 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
```
Comment (bpytlik) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:23]: It seems like this line should go away? Alternatively, if the changes in this function aren't really part of this PR, it probably makes sense to remove them.

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 2]
```diff
@@ -15,9 +15,45 @@
...
+
+
+
+  }
+
+  private ArrayList<TimeRange> FindIncompatibleRanges(Collection<Event> events, Collection<String> meetingattendees){
+      //Returns a list of times which a meeting cannot occur
+      ArrayList<TimeRange> incompatable-ranges = new Array<TimeRange>();
+      for(Event event : events){
+          TimeRange event-range = event.getWhen();
```
Comment (bpytlik) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:49]: nit: Using `-` in a variable name is unusual (to the point where I'm surprised it even compiles, but I learn new things every day). Please use `_` or camel case which would look like this: `eventRange`.

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 3]
```diff
@@ -15,9 +15,45 @@
...
+          Collection<String> event-attendees = event.getAttendees();
+          if (areMeetingAttendeesInEvent(meetingattendees,event-attendees)){
```
Comment (bpytlik) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:51]: Maybe I'm being dense, but I don't see the `areMeetingAttendeesInEvent` function defined anywhere.

Comment (bpytlik) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java:51]: Between these two lines are an example of where a 'why' comment might be helpful. For example something like: "If a previously scheduled event has at least one attendee of group we're scheduling for, then the time range for that meeting must be excluded, so include it in the result."

**Comment by CesarRiv** [comment]
Closing branch because committed a file that should not be here. Followed guides and was unsuccessful on trying to un-commit the file "DataServlet.Java"

**Pull request closed without merging**